packages/components/rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import json from '@rollup/plugin-json';
+import babel from '@rollup/plugin-babel';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import postcss from 'rollup-plugin-postcss';
+import resolve from '@rollup/plugin-node-resolve';
+import sassFunctions from 'sass-functions';
+import sassGlobImporter from 'node-sass-glob-importer';
+import del from 'rollup-plugin-delete';
+import path from 'path';
+import sass from 'sass';
+// eslint-disable-next-line import/extensions
+import pkg from './package.json';
+
+export default {
+  input: 'src/index.jsx',
+  output: [
+    {
+      file: pkg.module,
+      format: 'es',
+      sourcemap: true,
+    },
+    {
+      file: pkg.main,
+      format: 'cjs',
+      sourcemap: true,
+    },
+  ],
+  plugins: [
+    del({
+      targets: path.join(pkg.module, '../*'),
+    }),
+    peerDepsExternal(),
+    postcss({
+      extract: true,
+      modules: true,
```

Comment:
don't we need the classnames magic here same as in the core repo?

Reply:
No I wouldn't use the long fancy class names for the finished components build. They can't be overwritten anyway. Would rather spend time to keep component names in devtools

Preview is something different - there it could be useful.